### PR TITLE
Rename and version number fix ETT ckan files

### PR DESCRIPTION
**Pull request by politas**
Fix version sorting

### CODE REVIEW

**Comment by plague006** [comment]
As https://github.com/KSP-CKAN/CKAN-meta/pull/860 changing filenames will have no effect.

File: ETT/ETT-1.0.2_20150904.ckan [hunk 1]
```diff
@@ -10,7 +10,7 @@
         "kerbalstuff": "https://kerbalstuff.com/mod/901/Engineering%20Tech%20Tree",
         "x_screenshot": "https://kerbalstuff.com/content/Probus_200/Engineering_Tech_Tree/Engineering_Tech_Tree-1433986841.3047912.jpg"
     },
-    "version": "Version_1.0.2_20150904",
+    "version": "1.0.2_20150904",
```
Comment (Olympic1) [inline ETT/ETT-1.0.2_20150904.ckan]: should replace the `_` with a `.` as you replaced that in the other file

Comment (Olympic1) [inline ETT/ETT-1.0.2_20150904.ckan]: Should replace the `_` with a `.` as you replaced that in the other file. Same for the filename.

Comment (politas) [inline ETT/ETT-1.0.2_20150904.ckan]: No, because this is version 1.0.2, not version 1.0(.0) It's a weird inconsistency, but it should sort correctly.

**Comment by Olympic1** [comment]
Ah ok then. Merging with my thanks

**Comment by plague006** [comment]
I understand @Olympic1's point of view you're making the version formatting even more inconsistent. Instead of changing `"version": "1.0_20150731",` to `"version": "1.0.20150731",` why not change `ETT-1.0.20150622` to `ETT-1.0_20150622`. This will maintain the author's consistent standard of ksp_version_date

**Comment by Olympic1** [comment]
oops, merged the moment you commented this

**Comment by plague006** [comment]
Yeah, funny timing. Will make a new PR shortly.

**Comment by politas** [comment]
This is exactly how it was - which didn't sort correctly. _ sorts after ., so 1.0_20 comes after 1.0.2_20.

**Pull request merged**